### ISSUE TITLE
Toggleable not working on field "fieldset" 

### ISSUE THREAD

**Issue by rcheetah** [getgrav/grav-plugin-admin#2040]
[According to the docs](https://learn.getgrav.org/17/forms/blueprints/fields-available#fieldset-field), the field `fieldset` allows the common attribute `toggleable`. This took me by surprise, but would actually be ideal for my case.

However the setting does not seem to work. If I add it to my fieldset nothing changes. There is no checkbox to activate or deactivate the fieldset. Is toggleable actually implemented for fieldsets, or is this a mistake in the docs?

I think this would be a great option, because it would be an easy option to activate or deactivate a whole bunch of correlating settings.

**Comment by rhukster** [comment]
Could you please provide a sample of your YAML blueprint where you are doing this? It will help us recreate the issue you are seeing.

**Comment by rcheetah** [comment]
Yeah, okay. Here is a default.yaml (please rename) that adds two fieldsets, one that is toggleable, one regular.

[grav-plugin-admin #2040.yaml.zip](https://github.com/getgrav/grav-plugin-admin/files/5920476/grav-plugin-admin.2040.yaml.zip)

**Comment by rhukster** [comment]
It seems it does not support toggleable at all, i think it's a mistake in the docs i'm afraid.

**Comment by mahagr** [comment]
Fieldset is not a field by itself and such it cannot be toggleable.
